zimlii/templates/peachjam/_footer.html

```diff
@@ -0,0 +1,82 @@
+{% extends "peachjam/_footer.html" %}
+{% load i18n static %}
+{% block newsletter-form-code %}
+  <form action=""
+        method="post"
+        name="mc-embedded-subscribe-form"
+        target="_blank">
+    <div style="position: absolute; left: -5000px;" aria-hidden="true">
+      <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+      <input type="text" name="" tabindex="-1" value=""/>
+    </div>
+    <div class="input-group">
+      <input type="email"
+             value=""
+             name="EMAIL"
+             class="form-control"
+             placeholder="email address"
+             required=""/>
+      <input type="submit"
+             value="Subscribe"
+             name="subscribe"
+             class="btn btn-secondary"/>
+    </div>
+  </form>
+{% endblock %}
+{% block about %}
+  <p>
+    {% blocktrans trimmed %}
+    The Zimbabwe Legal Information Institute (ZimLII) is an independent not-for-profit Trust hosting an online repository of legal information from Zimbabwe and beyond.
+    ZimLII aims to provide for the knowledge needs of a growing group of people learning, aware and interested in the justice and legal framework of our country.
+    ZimLII is a member of the global Free Access to Law Movement and works closely with the African and international free access to law community to promote justice and rule of law by maximising access at no cost to public legal information.
+    {% endblocktrans %}
+  </p>
+  <div>
+    <a href="{% url 'terms_of_use' %}">{% trans 'Terms of Use' %}</a>
+  </div>
+{% endblock %}
+{% block second-content-column %}
+  <h5>{% trans 'Our partners' %}</h5>
+  <div>
+    <a href="https://agp.africanlii.org/" target="_blank" rel="noreferrer">AfricanLII</a>
+  </div>
+  <div>
+    <a href="https://laws.africa/" target="_blank" rel="noreferrer">Laws.Africa</a>
+  </div>
+  <div>
+    <a href="https://lrfzim.com/" target="_blank" rel="noreferrer">{% trans 'Legal Resources Foundation' %}</a>
+  </div>
+  <h6 class="mt-4">{% trans 'With support from' %}</h6>
+  <div>
+    <a href="https://sida.se/" target="_blank" rel="noreferrer">{% trans 'SIDA' %}</a>
```

Comment:
@niiroobiro could you confirm if the above url for SIDA is correct? `https://sida.se`

Reply:
@actlikewill Yeah..Confirmed.That's the correct URL.